paddle/operators/adam_op.cc

```diff
@@ -121,10 +113,8 @@ Adam updates:
 
 moment1_out = beta1 * moment1 + (1 − beta1) * grad
 moment2_out = beta2 * moment2 + (1 − beta2) * grad * grad
-beta1_pow_out = beta1_pow * beta1
-beta2_pow_out = beta2_pow * beta2
 learning_rate_t = learning_rate_t *
-                  sqrt(1 - beta2_pow_out) / (1 - beta1_pow_out)
+                  sqrt(1 - beta2_pow) / (1 - beta1_pow)
```

Comment:
Looks like we no longer need to update `beta_pow` to `beta_pow * beta` inside the `operatorKernel`. I am wondering when and where will `beta_pow` be updated?

Reply:
We do not need these because beta1 and beta2 power accumulators will be constant for all parameters. On the other hand, we will have one instance of the adam_op for every parameter. So these will be done in accumulators that the Python API will control.

Reply:
Great.